### PR TITLE
Add me.acristoffers.Lachesis

### PR DESCRIPTION
**Pull request by acristoffers**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). (see https://acristoffers.me/projects/lachesis)
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: me.acristoffers.Lachesis.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        url: https://github.com/acristoffers/Lachesis.git
+        branch: master
+        commit: b507c3475d8ec3460df846fa9dc26163ff799369
+      - generated-sources.json
+      # Wrapper to launch the app
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/main/lachesis "$@"
+      - type: dir
```
Comment (hfiguiere) [inline me.acristoffers.Lachesis.yml:47]: 1. don't use dir
2. we only need ONE size. 128x128. 256x256 can also do it. Everything else is unucessary and should instead be ALL managed upstream.

File: me.acristoffers.Lachesis.desktop [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=Lachesis
+Comment=Moirai supervisor
+Categories=Science;Engineering;
+
+Icon=me.acristoffers.Lachesis
+Exec=/usr/bin/flatpak --nosocket=fallback-x11 --socket=x11 run me.acristoffers.Lachesis
```
Comment (hfiguiere) [inline me.acristoffers.Lachesis.desktop:10]: this is wrong. it should just be the executable name, maybe with command line argument substitution

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing.

File: me.acristoffers.Lachesis.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      - install -Dm644 icon_128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon_256x256.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon_512x512.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      - install -Dm755 -t /app/bin/ run.sh
+    sources:
+      - type: git
+        url: https://github.com/acristoffers/Lachesis.git
+        branch: master
```
Comment (hfiguiere) [inline me.acristoffers.Lachesis.yml:39]: NEVER branch
```suggestion
```

File: me.acristoffers.Lachesis.yml (hunk 1 above)
Comment (acristoffers) [reply me.acristoffers.Lachesis.yml:47]: Ok, I'll change it to have only the 256x256 then.

File: me.acristoffers.Lachesis.desktop (hunk 2 above)
Comment (acristoffers) [reply me.acristoffers.Lachesis.desktop:10]: Wait, what? If I just put lachesis there it doesn't work. Or should it be the run.sh script? Dropbox'es desktop file also does it like that.

File: flathub.json (hunk 3 above)
Comment (acristoffers) [reply flathub.json:2]: Not that I think about it, it's not necessary either. Electron IS available in other platforms...

File: me.acristoffers.Lachesis.yml (hunk 4 above)
Comment (acristoffers) [reply me.acristoffers.Lachesis.yml:39]: I'll remove it, it was in the flatpak example.

File: me.acristoffers.Lachesis.desktop (hunk 2 above)
Comment (hfiguiere) [reply me.acristoffers.Lachesis.desktop:10]: it should be the same than a in `command`.

**Comment by acristoffers** [comment]
bot, build me.acristoffers.Lachesis

**Comment by acristoffers** [comment]
The build is working alright.

**Comment by acristoffers** [comment]
bot, build me.acristoffers.Lachesis

**Pull request closed without merging**